server/src/main/resources/db/migration/V1_324__create_table_duolingo_coupons.sql

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2024 Talent Beyond Boundaries.
+ *
+ * This program is free software: you can redistribute it and/or modify it under
+ * the terms of the GNU Affero General Public License as published by the Free
+ * Software Foundation, either version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see https://www.gnu.org/licenses/.
+ */
+
+-- Create the coupon table
+CREATE TABLE duolingo_coupon (
+                         id BIGINT DEFAULT nextval('coupon_id_seq') PRIMARY KEY,
+                         coupon_code VARCHAR(255) UNIQUE NOT NULL,
+                         candidate_id BIGINT,
+                         expiration_date TIMESTAMP NOT NULL,
+                         date_sent TIMESTAMP,
+                         assignee_email VARCHAR(255),
```

Comment:
I think that it is potentially confusing to have this email as well as the email associated with the candidate. I would remove this field.